vNext/channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -136,35 +157,26 @@ export abstract class EnvelopeCreator {
 
         if (item.ext.os) {
             if (item.ext.os.deviceOS) {
-                env.tags[CtxTagKeys.deviceOS] = item.ext.os.deviceOS;
+                env.tags[CtxTagKeys.deviceOS] = item.ext.os.name;
             }
         }
 
-        if (item.tags[LegacyKeys.deviceNetwork]) {
-            env.tags[CtxTagKeys.deviceNetwork] = item.tags[LegacyKeys.deviceNetwork];
-        }
         if (item.ext.device) {
             if (item.ext.device.deviceType) {
                 env.tags[CtxTagKeys.deviceType] = item.ext.device.deviceType;
             }
         }
 
-        if (item.ext.web) {
-            if (item.ext.web.screenRes) {
-                env.tags[CtxTagKeys.deviceScreenResolution] = item.ext.web.screenRes;
-            }
-        }
-
-        if (item.tags[SampleRate]) {
-            env.tags.sampleRate = item.tags[SampleRate];
-        }
-        
         // No support for mapping Trace.traceState to 2.0 as it is currently empty
 
         if (item.ext.trace) {
             if (item.ext.trace.parentID) {
                 env.tags[CtxTagKeys.operationParentId] = item.ext.trace.parentID;
             }
+
+            if (item.ext.trace.name) {
```

Comment:
What is this property and how is used?

Reply:
I did not remove this, kept this mapping in for operation.name as it is auto captured in AI SDK today and I have not received confirmation that UI will work without it.